### PR TITLE
docs: use `Deno.serve`

### PR DESCRIPTION
**Pull request by idanran**

### CODE REVIEW

**Review by lino-levan — APPROVED** [review]
LGTM. One minor nit.

**Review by crowlKats — APPROVED** [review]
LGTM, thanks

File: README.md [hunk 1]
```diff
@@ -78,9 +78,7 @@ deno run https://deno.land/std/examples/welcome.ts
 Or [setup a simple HTTP server](https://examples.deno.land/http-server):
 
 ```ts
-import { serve } from "https://deno.land/std@0.182.0/http/server.ts";
-
-serve((_req) => new Response("Hello, World!"));
+Deno.serve((_req) => new Response("Hello, World!"));
```
Comment (lino-levan) [inline README.md:81]: Do we want to keep `_req` here?

Comment (crowlKats) [reply README.md:81]: yes; it at least gives a general idea of what the first & most commonly used argument is without having to be specific

Comment (lino-levan) [reply README.md:81]: Makes sense. I thought that might be it.

**Pull request merged**